### PR TITLE
[Messenger] Allow to configure the db index on Redis transport

### LINKED ISSUE #33724: [Messenger] Redis transport should support db-index
**Description**  
As already mentioned in [Issue 32304](https://github.com/symfony/symfony/issues/32304) it would be fine if messenger redis transport could support configurable db-index. Actually i have db0 & db1 for another project and symfony cache already in db2. So it would be fine to have the messenger queue also in db2.

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | Fixes #33724 
| License       | MIT
| Doc PR        | todo

Quite useful for testing.

### CODE REVIEW

**Comment by chalasr** [comment]
cc @alexander-schranz FYI

**Review by alexander-schranz — APPROVED** [review]
Looks good for me. Thank you  @chalasr 👍

File: src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php [hunk 1]
```diff
@@ -31,7 +31,8 @@ class Connection
         'group' => 'symfony',
         'consumer' => 'consumer',
         'auto_setup' => true,
-        'stream_max_entries' => 0, // any value higher than 0 defines an approximate maximum number of stream entries
+        'stream_max_entries' => 0, // any value higher than 0 defines an approximate maximum number of stream entries,
```
Comment (fabpot) [inline src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php]: the added comma is probably not needed at the end of the comment

**Review by xabbuh — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @chalasr.

**Pull request merged**